src/ir/var.rs

```diff
@@ -199,6 +199,8 @@ impl ClangSubItemParser for Var {
                             true,
                             ctx,
                         );
+                        ctx.parse_callbacks()
```

Comment:
nit: Maybe `if let Some(callbacks) = ctx.parse_callbacks() {` for consistency with some of the other branches than don't return any value? I don't feel very strongly about it in any case.